Source/Core/VideoCommon/GraphicsModSystem/Runtime/GraphicsModManager.cpp

```diff
@@ -169,19 +169,19 @@ void GraphicsModManager::Load(const GraphicsModGroupConfig& config)
     {
       const auto create_action = [](const std::string_view& action_name,
                                     const picojson::value& json_data,
-                                    GraphicsModConfig mod) -> std::unique_ptr<GraphicsModAction> {
+                                    GraphicsModConfig mod_) -> std::unique_ptr<GraphicsModAction> {
```

Comment:
Can you change this one to be `mod_config`?  Calling it `mod` in the first place was bad on my part.  I am about to have a change to this part of the code, so I'm happy to change it then too.

Reply:
Done.